### ISSUE TITLE
Refactor search filter path literals to constant values

### ISSUE THREAD

**Issue by rhysyngsun** [mitodl/micromasters#4001]
We define these as inline literals wherever we use them in JS, meaning if we miss one usage we can easily trigger a regression. We should define constants for all the keys in `SEARCH_FACET_FIELD_LABEL_MAP` and update all usages of those key strings to also use the constants as well. That way we only need to change the filter name in one place to update all our JS.

**Comment by pdpinch** [comment]
I don't think this is actively being worked on. I'm moving it back to the backlog.
